app/services/csv_service.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.